src/WWT.Providers/Providers/DemMarsEmptyProvider.cs

```diff
@@ -34,7 +35,7 @@ public override void Run(IWwtContext context)
 
                     WebClient webclient = new WebClient();
 
-                    string url = string.Format("http://wwt.nasa.gov/wwt/p/mars_toast_dem_32f/{0}/{1}/{2}.toast_dem_v1", level, tileX, tileY);
+                    string url = $"http://wwt.nasa.gov/wwt/p/mars_toast_dem_32f/{level}/{tileX}/{tileY}.toast_dem_v1";
```

Comment:
FYI, it looks like `wwt.nasa.gov` no longer exists. So any of the providers that try to pull backing data from the server, like this one, are going to be broken. I need to analyze these and see if there are any that are important to rescue.